portfolio/src/main/webapp/script.js

```diff
@@ -39,6 +39,27 @@ function addQuote() {
 
 /** Display comments on the page, capped at the user's requested maximum */
```

Comment:
What if the datastore is empty? What if the user's requested maximum is greater than the actual number of comments?

Reply:
If the datastore is empty, then no comments are posted. If the requested max is greater, then all comments are shown